tools/templates/tools/usertool_create.jinja

```diff
@@ -1,12 +1,9 @@
 {% extends "layouts/base.jinja" %}
 
-{% block page_title %}
-    Create a tool
+{% block head_title %}
+    {{ _("Create a tool") }}
 {% endblock %}
 
 {% block content %}
     {{ crispy(form) }}
```

Comment:
The taxonomies widget needs work. Trying to type `Hand tools/hammers` creates a separate `hand` tag before I can complete typing the rest. Also, add some help text so the user know what the format should be for nesting. Maybe some extra explanation copy for what clearance and visibility means.